test/packages/basic_cpp/Doxyfile

```diff
@@ -0,0 +1,21 @@
+# All settings not listed here will use the Doxygen default values.
```

Comment:
Did this leak in from another change?

Reply:
No, but it is only weakly related to this PR. This PR deals with when Doxygen is run. I did not have any tests that explicitly added a Doxyfile, so I added one here to make sure that things still worked.